src/downshift.js

```diff
@@ -1090,16 +1115,19 @@ function validateGetRootPropsCalledCorrectly(element, {refKey}) {
   const refKeySpecified = refKey !== 'ref'
   const isComposite = !isDOMElement(element)
   if (isComposite && !refKeySpecified) {
-    throw new Error(
+    // eslint-disable-next-line no-console
+    console.error(
       'downshift: You returned a non-DOM element. You must specify a refKey in getRootProps',
     )
   } else if (!isComposite && refKeySpecified) {
-    throw new Error(
+    // eslint-disable-next-line no-console
+    console.error(
       `downshift: You returned a DOM element. You should not specify a refKey in getRootProps. You specified "${refKey}"`,
     )
   }
   if (!getElementProps(element)[refKey]) {
-    throw new Error(
+    // eslint-disable-next-line no-console
```

Comment:
Let's disable the `no-console` rule entirely. It's kinda silly at this point.

Thanks!

Reply:
IMHO it still makes sense to have this rule and disable it on per call site basis - this guards from unintended logs being shipped by accident, but still allows for granular control for cases when we actually want those console logs

at least this is what I always do at work

let me know what do you think about it

Reply:
That's fine with me 👌